source/_integrations/nest.markdown

```diff
@@ -191,7 +191,11 @@ script:
 
 ### Troubleshooting
 
-- If you're getting [rickrolled](https://www.youtube.com/watch?v=dQw4w9WgXcQ) instead of being able to see your Nest cameras, you may not have set up your developer account's permissions correctly. Go back through and make sure you've selected read/write under every category that it's an option.
+- See [No URL Available](https://www.home-assistant.io/more-info/no-url-available) for guidance on setup issues related to URLs.
```

Comment:
The more info pages are meant for linking from the UI and will be served to the user when appropriate.

```suggestion
```